### ISSUE TITLE
Gamification Service hexagon no longer available in Bluemix ... it seems so ... 

### ISSUE THREAD

**Issue by thomassuedbroecker** [IBM-MIL/IBM-Ready-App-for-Venue#12]
Hello,
the requested services at  _Step 4: Create Gamification Service Instance_  in the http://lexdcy040194.ecloud.edst.ibm.com/venue_1_0_0/getting_started is no longer available in Bluemix called **"Gamification hexagon"**.
Which other service can be used?
Regards,
Thoma

**Comment by bweagle** [comment]
Perhaps this service got moved around. I was able to find it in Bluemix here: https://console.ng.bluemix.net/catalog/services/gamification/

**Comment by ynchan** [comment]
I can't find it either, looks like the service has been removed.

**Comment by jimmyliao** [comment]
The IBM Gamification service is being discontinued.
https://twitter.com/ibmcloudsupport/status/704688681613959168

**Comment by craigkj312** [comment]
Correct. The Gamification service has been discontinued, but the Ready App should be able to function without a connection to it.
